NuRadioReco/modules/io/eventWriter.py

```diff
@@ -5,6 +5,7 @@
     import pickle
```

Comment:
If I am not mistaken, python 3 has no cPickle anymore. We can remove these calls.